test/e2e/storage/testpatterns/testpattern.go

```diff
@@ -162,6 +186,13 @@ var (
 		FsType:     "ntfs",
 		FeatureTag: "[sig-windows]",
 	}
+	// NtfsEphemeralVolume is TestPattern for "Ephemeral-volume (ntfs)"
+	NtfsEphemeralVolume = TestPattern{
```

Comment:
csi is not supported on windows yet. This will probably break windows tests?

Reply:
I just added it for the sake of completeness. It's not used anywhere yet.